plutus-core/plutus-core/src/Universe/Core.hs

```diff
@@ -432,7 +432,7 @@ someValueOf :: forall a uni. uni (Esc a) -> a -> Some (ValueOf uni)
 someValueOf uni = Some . ValueOf uni
 
 -- | Wrap a value into @Some (ValueOf uni)@, provided its type is in the universe.
-someValue :: forall a uni. uni `Includes` a => a -> Some (ValueOf uni)
+someValue :: forall a uni. uni `Contains` a => a -> Some (ValueOf uni)
```

Comment:
Because in this case `Includes` expands to `Contains` (this is always the case when the kind of its second argument is `*`), so I thought it was worth removing the indirection.